api-reference/10 UI Widgets/GridBase/1 Configuration/rowDragging/allowDropInsideItem.md

```diff
@@ -8,8 +8,9 @@ default: false
 Allows users to drop a row inside another row.
```

Comment:
It looks like this option should be available for TreeList only. Now it is available for DataGrid too.

Reply:
I consulted with the devs. @vconst told that this option should be available for the DataGrid too. A possible scenario - somebody creates a file manager using our DataGrid, and users can move files to a folder via drag and drop.

Reply:
No, It is available in DataGrid also.